src/sentry/event_manager.py

```diff
@@ -771,16 +774,14 @@ def get_data(self):
         return self._data
 
     def _get_event_instance(self, project_id=None):
-        data = self._data.copy()
-        event_id = data.pop('event_id')
-        platform = data.pop('platform', None)
+        data = self._data
+        event_id = data.get('event_id')
+        platform = data.get('platform')
 
-        recorded_timestamp = data.pop('timestamp')
+        recorded_timestamp = data.get('timestamp')
```

Comment:
Does it matter that the `event_id` and `timestamp` getters are becoming safe? They would throw a `KeyError` before, and I'm not sure if that would catch anything of value before it caused problems.

Reply:
There is no access to these attributes that would put stuff in.